test/JDBC/upgrade/latest/schedule

```diff
@@ -529,3 +529,4 @@ babel_test_int8_numeric_oper
 babel_test_int2_numeric_oper
 login_token-dep
 test_like_for_AI
+BABEL-4869
```

Comment:
Why only latest schedule? I don't see anything in prepare file which is not supported in the previous versions.

Reply:
Updated